### ISSUE TITLE
Switch to only VueUse focus trap component.

### ISSUE THREAD

**Issue by patrickcate** [patrickcate/vue-uswds#437]
Now that VueUse's focus trap component allows for options to be passed to the focus trap library, all non-component uses of the composable should be able to be switched to the component version.
